tests/component/system/test_device.py

```diff
@@ -35,5 +35,9 @@ def test___self_test_device___no_errors(any_x_series_device: Device) -> None:
     any_x_series_device.self_test_device()
 
 
-def test___self_cal___no_errors(any_x_series_device: Device) -> None:
-    any_x_series_device.self_cal()
+def test___restore_last_ext_cal_const___no_errors(sim_x_series_device: Device) -> None:
```

Comment:
> nidaqmx.errors.RpcError: StatusCode.UNIMPLEMENTED: This operation is not supported by the NI gRPC Device Server being used. Upgrade NI gRPC Device Server.

Add grpc_xfail to this test:
```suggestion
@pytest.mark.grpc_xfail(
    reason="Requires NI gRPC Device Server version 2.5 or later", raises=RpcError
)
def test___restore_last_ext_cal_const___no_errors(sim_x_series_device: Device) -> None:
```

Reply:
Thanks for the suggestion, I have marked the test with grpc_xfail as per suggested.